regexes/device/mobiles.yml

```diff
@@ -14222,15 +14331,19 @@ Wiko:
       model: 'View 2'
     - regex: 'W_P200'
       model: 'View Max'
+    - regex: 'W_P130'
+      model: 'View Go'
     - regex: 'W-V800-(TVM|EEA|OPE)'
       model: 'View 3 Lite'
-    - regex: 'W-K420-EEA'
+    - regex: 'W-K130-(?:TMV|OPE|EEA)'
       model: 'Y50'
-    - regex: 'W-K510-(TVM|EEA|OPE|SUN)'
+    - regex: 'W-K420-EEA'
```

Comment:
According to a quick search `W-K420-EEA` is actually the `Y50` as it was defined before. Where did you have the information from that it's `Y70`?

Reply:
I used the official website https://world.wikomobile.com/red.php?telephone=2725

Reply:
ok fine. Guess the search results on Google are a bit confusing than...